zerver/openapi/zulip.yaml

```diff
@@ -8261,6 +8266,89 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+  /user_topics:
+    post:
+      operationId: update-user-topic
+      summary: Update visibility policy of a topic
+      tags: ["streams"]
+      description: |
+        This endpoint is used to update the visibility policy of a topic.
+
+        - 0 - INHERIT
+        - 1 - MUTED
+        - 2 - UNMUTED
+
+        The visibility policy, when set to MUTED, mutes a topic;
+        when set to UNMUTED, it unmutes a topic in a muted stream;
+        and INHERIT is used to remove the visibility policy already set.
+
+        This endpoint mutes or unmutes a topic in a stream or unmutes a topic
+        in a muted stream that the current user is subscribed to.
+        MUTED topics are displayed faded in the Zulip UI and are not included
+        in the user's unread count totals. An UNMUTED topic in a muted stream
+        behaves as an unmuted topic in an unmuted stream.
```

Comment:
The `visibility_policy` documentation belongs in the description of the field itself, not in the endpoint's description, which should just explain that this modifies for the visibility policy for the single stream and topic pair indicated by the parameters.

Reply:
Makes sense -- updated!